### DISCUSSION TITLE
Dots in column names of a Form

### DISCUSSION THREAD

**Discussion by apcjs** [mantinedev/mantine#6254] [Q&A]
I have my initialValues as follows

```
{
  "Company.Name": "Test",
  "FirstName": "Arun"
}

```
`form.setFieldValue("Company.Name", "Test2")` throws an error since Form expects Name to be a child JSON of Company. Is there any way to let the Form know that this column is not a child?

**Comment by rtivital** [comment]
No, dot meaning cannot be changed

**Comment by apcjs** [comment]
I did notice that when I went through the source code.

In get-splitted-path.ts, can we not `return [path]` if the path starts and ends with a double quote?

Then I can use `form.setFieldValue('"Company.Name"', "Test2")`
